src/main/java/util/events/ShowIssuePickerEvent.java

```diff
@@ -0,0 +1,23 @@
+package util.events;
+
+import java.util.List;
+
+import backend.resource.TurboIssue;
+
+/**
+ * An event to show Issue Picker 
+ */
+public class ShowIssuePickerEvent extends Event {
+
+    public final boolean isStandalone;
+    public final List<TurboIssue> allIssues;
+    
+    /**
+     * @param isStandalone determines if the caller expects a return result
+     * e.g when triggered by other UI element such as the filter text field
```

Comment:
I'm not sure what this means. It doesn't seem to be used either. As opposed to getting a result through a side effect?